wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add card template with tabbed views

Fixes #10635 

This adds a card template that will be used for cards with tabbed data (i.e. Insights Comments and Insights Followers).

View components include:
- `FilterTabBar` to switch between data views.
- Total count.
- Subtitles.
  - Subtitles and total count hide if there is no data.
- Data rows.
  - The disclosure does nothing yet.
- `View more` if the number of data rows exceeds 6.
  - This shows a temporary alert.
- `No data yet` if there is no data.

To test:
- Go to Stats > Insights.
- Scroll to the Comments card. Since this is just a template, the data shown is purely fake.
- Switch between the tabs. Verify the views update accordingly.

![tabs](https://user-images.githubusercontent.com/1816888/49904439-5cdd8080-fe27-11e8-9047-bf0495b859d4.png)

---
To see the different states, you can modify the parameters sent when creating the data for the card. It's all done in `SiteStatsInsightsViewModel:createTabbedTotalsStatsRow`.

To hide the total count label, modify the `TabbedTotalsStatsRow:showTotalCount` parameter.

<img width="350" alt="no_total_count" src="https://user-images.githubusercontent.com/1816888/49904193-b42f2100-fe26-11e8-9f98-3c4ed06a1ca9.png">

To see `No data yet`,  send an empty array for the `TabData:dataRows` parameter.

<img width="350" alt="no_data" src="https://user-images.githubusercontent.com/1816888/49904299-f5273580-fe26-11e8-9fe1-565ca9849d1e.png">
